### ISSUE TITLE
Missing styles after uploading to GraphSpace

### ISSUE THREAD

**Issue by LHuang2019** [Murali-group/CyGraphSpace#13]
1. The issue is found by exporting the network generated by PathLinker to GraphSpace. The result graph does not apply the correct background color (yellow/light-blue in this case) mapping for the source and the target nodes. The CyGraphSpace code base on master branch does export the style correctly. The issue only exists with the code base in the develop branch.
 
2. Both master and develop branch fails to apply the hierarchical layout to the PathLinker network.

![image](https://user-images.githubusercontent.com/14338820/36636101-768f4650-198d-11e8-87d8-a74b1667465a.png)

The above pictures comes from the CyGraphSpace develop branch. Both the color and the hierarchical layout are missing for the result PathLinker network.

**Comment by jlaw9** [comment]
You're right @LHuang2019. Just found that this is a known issue in cytoscape's [complete list of compatibilities](https://docs.google.com/spreadsheets/d/1iTDKGjro1-L7HGz1ftozqDKRxjQVSPF28jQLt7XGrSg/edit#gid=0) spreadsheet. I think there is a possible workaround here if the user unlocks the node/width height and then sets a mapper to the node degree themselves.

Another possible option would be to detect these types of bypasses, figure out what the actual style values are (node width and height in this case) and set them explicitly in the JSON being exported. Not sure if that would be possible though...

**Comment by jlaw9** [comment]
One more related issue is the graph's background color, which needs to be set in the CSS of the website. This is already an issue on GraphSpace: https://github.com/Murali-group/GraphSpace/issues/291

Once that issue is fixed, it should be simple to add the JSON field that would be recognized by GraphSpace.
